demo/prepare_xpack.py

```diff
@@ -0,0 +1,46 @@
+from elasticsearch_xpack import XPackClient
+import getpass
+import sys
+from utils import Elasticsearch
+
+if __name__ == "__main__":
+    if len(sys.argv) == 2:
+        password = getpass.getpass()
+    elif len(sys.argv) == 3:
+        password = sys.argv[2]
+    else:
+        sys.exit(-1)
+
+    username = sys.argv[1]
+
+    es = Elasticsearch(http_auth=(username, password))
+    xpack = XPackClient(es)
+
+    print("Configure ltr_admin role:")
+    res = xpack.security.put_role('ltr_admin', {
+        "cluster": ["all"],
+        "indices": [ {
+            "names": [".ltrstore*"],
+            "privileges": ["all"]
```

Comment:
I wonder if we can provide probably more find grain privileges? not sure we need all, what do you think about something like: manage, monitor, read and write?

Reply:
I didn't here as it was just the demo, but if we want to provide roles that we suggest to use on production clusters then certainly "all" is a bit too much. Also we should probably have separate `ltr_admin` and `ltr_query` roles, where `ltr_query` only needs read access to the index for when it needs to fetch a model not in cache. 

Its not particularly complicated so it probably makes sense to put the full featured setup here in the demo. Or even outside the demo and referenced from here? Not sure.

Reply:
I would probably add this as reference in the documentation.